### PR TITLE
templates: addition of settings templates

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* NEW Adds templates for the settings page. Modules can use these to
  include their own settings views in one common location.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by lnielsen** [comment]
Looks good, but can you have a look at the travis failure. Seems to just be import sorting problems.

**Comment by lnielsen** [comment]
Also, can you add a test case for template blocks. See example at https://github.com/inveniosoftware/invenio-theme/blob/master/tests/test_invenio_theme.py#L87

This ensures that accidentally renamed template blocks will be caught.

File: invenio_theme/templates/invenio_theme/settings/helpers.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+{%- block panel_start scoped %}
+<div class="panel-group panel-bot-margin"{% if id %} id="{{id}}"{% endif %}>
+    <div class="panel {{panel_extra_class}}">
+        {%- block panel_heading scoped %}
+        <div class="panel-heading">
+            <strong>{%if icon %}<i class="{{icon}}"></i>{% endif %} {{ title|safe }}</strong>
+            {%- if btn and (btn_href or btn_text) -%}
+            <div class="pull-right" style="margin-left: 5px;">
+            {%- if btn_text %}
+                <small class="text-muted" style="padding-right: 10px;">(updated {{last_sync}})</small>
```
Comment (lnielsen) [inline invenio_theme/templates/invenio_theme/settings/helpers.html]: I don't think this should be here.

**Comment by lnielsen** [comment]
Also, can you add a simple example of the menu to the example app, so that it's possible to see how it looks like.

**Comment by SamiHiltunen** [comment]
I fixed the issues @lnielsen  brought up. Tests should pass as soon as inveniosoftware/flask-menu#39 is released.

**Comment by lnielsen** [comment]
:shipit: Pending inveniosoftware/flask-menu#44

**Comment by lnielsen** [comment]
Merged via #14

**Pull request closed without merging**